### ISSUE TITLE
Could you add live reloading?

### ISSUE THREAD

**Issue by liquidvisual** [Splode/jekyll-vue-template#5]
Thanks for making this, great library! Is it possible to add some sort of live reloading - like BrowserSync?

**Comment by Splode** [comment]
Thanks, that's a great suggestion. I'm working on live reloading using Browsersync now. I have a working solution for live reloads triggered either by Jekyll or webpack-managed content. 

Unfortunately, this still requires manually running two separate shells, one for webpack and one for Jekyll, due to a race condition. Specifically, resources built with `webpack --watch` are unavailable at the time `bundle exec jekyll build --watch` is called, causing an error.

I'll continue looking into a more elegant solution, but for the time being this feature is available in the `dev` branch.

**Comment by liquidvisual** [comment]
Nice, thanks for that - I'll check it out!

I've also been playing with [vue-jekyll](https://github.com/BryanSchuetz/vue-jekyll), have you ever used it? If so, what is your opinion of it?

It seems to work quite well (single shell), although its dependencies are very much out of date. I was able to get the Sass working with BrowserSync after adding in the loaders.

**Comment by Splode** [comment]
Yep 👍🏻 In fact, Bryan Schuetz (creator of `vue-jekyll`) wrote an article, [Vue and Jekyll](https://distresssignal.org/vue-and-jekyll), which was the inspiration for this project. I wanted something similar, but with updated libraries and some features like code-splitting, hashing, etc.

**Comment by sonnienguyen** [comment]
I use the `npm-run-all` npm package, `jekyll serve --livereload` and `webpack —watch` to automatically refresh changes.

```diff
  "scripts": {
    "dev": "cross-env NODE_ENV=development webpack",
    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
+   "serve": "bundle exec jekyll serve --livereload",
+   "start": "run-p watch serve",
    "watch": "cross-env NODE_ENV=development webpack --watch"
  },

```

```diff
# _config.yml

  # Build settings
  markdown: kramdown

+ # Server settings
+ host: 0.0.0.0
+ port: 9001
+ encoding: UTF-8

```
